Airports/EGNO/VRPs.txt

```diff
@@ -1,5 +1,4 @@
 *Garstang Marina:N053.54.19.500:W002.47.30.200
-*M55 J3:N053.48.25.000:W002.53.23.000
```

Comment:
Per the VRP list, this VRP is 'Associated' with both Warton and Blackpool, so leave this definition in (our logic is to have them multiple-defined if they're associated with more than one AD).